src/ranch.erl

```diff
@@ -350,6 +353,11 @@ get_max_connections(Ref) ->
 set_max_connections(Ref, MaxConnections) ->
 	ranch_server:set_max_connections(Ref, MaxConnections).
 
+-spec get_transport(ref()) -> module().
+get_transport(Ref) ->
+	[_, Transport, _, _, _] = ranch_server:get_listener_start_args(Ref),
```

Comment:
I don't think we need to create a separate function tbh.

Reply:
Ok, done :)